README.md

```diff
@@ -63,7 +68,7 @@ Most of the events are selft explanatory, here's the ones that may need more det
 ####percentsPlayedInterval
 
 This options goes with the ```percentsPlayed``` event. Every ```percentsPlayedInterval``` percents an event will be sent to GA.
-**default:** 10
+**default:** 25
```

Comment:
in marketing you look after the 0-25 25-50 ... 
So that way i change you can keep the defaut